src/com/projectkorra/projectkorra/chiblocking/WarriorStance.java

```diff
@@ -15,17 +15,21 @@
 
 	@Attribute(Attribute.COOLDOWN)
 	private long cooldown;
+	@Attribute(Attribute.DURATION)
+	private long duration;

```

Comment:
An accessor `public long getDuration()` and mutator `public void setDuration(long duration)` should be added in this PR to account for the added `long duration` variable.